dev/devicelab/bin/tasks/flavors_test_macos.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_devicelab/framework/devices.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/tasks/integration_tests.dart';
+
+Future<void> main() async {
+  deviceOperatingSystem = DeviceOperatingSystem.macos;
+  await task(() async {
+    await createFlavorsTest().call();
+    await createIntegrationTestFlavorsTest().call();
```

Comment:
The file is missing `install` tests, see `dev/devicelab/bin/tasks/flavors_test.dart`.